mangaki/mangaki/tasks.py

```diff
@@ -0,0 +1 @@
+# Here goes the Celery tasks.
```

Comment:
Well, MAL import should be a Celery task, that's a sufficient and important use case.
As we also may support new sources of data for imports (e.g. AniDB).

Using this PR, I hacked a prototype of MAL import as a Celery task and it worked well, we could enable a better UX than `location.reload()` and hope everything works once the requests returns.

This also could serve as a basis for complex admin tasks (e.g. long merges, poster refresh, NSFW poster detection could be one of them).